CreeDictionary/CreeDictionary/settings.py

```diff
@@ -153,6 +153,35 @@
 
 ############################### Morphodict configuration ###############################
 
+MORPHODICT_SOURCES = [
```

Comment:
why it's moved here: It's reasonable to move DatabaseManager (importer) to morphodict eventually.

Reply:
Yeah, it was hacky to have this stuff in a migration! [checks `git blame`] whoops 🙃